pandas/core/series.py

```diff
@@ -2559,27 +2559,12 @@ def memory_usage(self, index=True, deep=False):
             v += self.index.memory_usage(deep=deep)
         return v
 
-    def take(self, indices, axis=0, convert=True, is_copy=False, **kwargs):
+    def _take(self, indices, axis=0, convert=True, is_copy=False):
         """
-        return Series corresponding to requested indices
-
-        Parameters
-        ----------
-        indices : list / array of ints
-        convert : translate negative to positive indices (default)
-
-        Returns
-        -------
-        taken : Series
-
-        See also
-        --------
-        numpy.ndarray.take
```

Comment:
why do we have this? now that it is generic is shouldn't be needed

Reply:
Have what exactly?